.gitlab-ci.yml

```diff
@@ -32,8 +32,10 @@ build:
   script: |
     VERSION=${CI_COMMIT_TAG:-$CI_COMMIT_SHA}
     echo 'let appVersion = "'${VERSION}'"' > ./Sources/App/Core/AppVersion.swift
+
+    docker run --rm -it -v $PWD:/host -w /host --entrypoint yarn node:15.8-alpine build
```

Comment:
need to drop `-it` since we're not running it from a terminal:

```
$ VERSION=${CI_COMMIT_TAG:-$CI_COMMIT_SHA} # collapsed multi-line command
the input device is not a TTY
```

https://gitlab.com/finestructure/swiftpackageindex/-/jobs/1033648182

Reply:
I removed it but didn't replace it with anything. Is that what's needed?

Reply:
Yep, that should do! I’ll verify if the build passed. I don’t think we should merge without confirming the deployment images are building correctly.

Reply:
Right, so this command isn’t quite standalone, is it? We need `parcel`?

```
Digest: sha256:db3f9c88223ba2491e7d2846a18efe46ecb4b60bf2bcf4b53cdcd39e5f6888b5
Status: Downloaded newer image for node:15.8-alpine
yarn run v1.22.5
$ parcel build Resources/Scripts/main.js --experimental-scope-hoisting -d Public
/bin/sh: parcel: not found
error Command failed with exit code 127.
```

https://gitlab.com/finestructure/swiftpackageindex/-/jobs/1034263910

Reply:
It should be standalone. `parcel` is a dependency in the `package.json` file which yarn should install.

Reply:
Oh, wait. I think I know what's needed.